diff -ruN zmx-orig/build.zig zmx/build.zig
--- zmx-orig/build.zig	2026-10-03 02:39:00.879857512 -0500
+++ zmx/build.zig	2026-10-03 02:39:00.882123054 -0500
@@ -20,7 +20,7 @@
 
     const options = b.addOptions();
     options.addOption([]const u8, "version", version);
-    const ghostty_ver = build_zig_zon.dependencies.ghostty.hash;
+    const ghostty_ver = "ghostty-1.3.2-dev-5UdBC4FtYAV-PdFOt-gceWieTHLri0H4ztHtd6Tr4dGj+agterm";
     options.addOption([]const u8, "ghostty_version", ghostty_ver);
 
     const exe_mod = b.createModule(.{
diff -ruN zmx-orig/build.zig.zon zmx/build.zig.zon
--- zmx-orig/build.zig.zon	2026-10-03 02:47:56.508894151 -0500
+++ zmx/build.zig.zon	2026-10-03 02:47:56.511145527 -0500
@@ -5,8 +5,7 @@
     .minimum_zig_version = "0.16.0",
     .dependencies = .{
         .ghostty = .{
-            .url = "git+https://github.com/ghostty-org/ghostty#8af6897c0afc63037a8a3efee4162a380e3a4572",
-            .hash = "ghostty-1.3.2-dev-5UdBC4FtYAV-PdFOt-gceWieTHLri0H4ztHtd6Tr4dGj",
+            .path = "../zmx-ghostty",
         },
     },
     .paths = .{
